requirements/build.txt

```diff
@@ -7,4 +7,4 @@ sphinx-rtd-theme
 sphinx-autobuild
 pex
 wheel
-setuptools<20.11,>=2.2          # needed to get pex working
+setuptools>=2.2,<20.11          # needed to get pex working # pyup: ignore
```

Comment:
does the double-comment work? (two `#`s)

Reply:
It's just in case they pattern recognize on `# pyup:` :) I've seen this problem with `# noqa` and flake8...